packages/gatsby-cli/src/reporter/prepare-stack-trace.ts

```diff
@@ -13,11 +13,14 @@ import {
 } from "source-map"
 
 export class ErrorWithCodeFrame extends Error {
-  codeFrame: string
+  codeFrame = ``
```

Comment:
Similar as below
```suggestion
  codeFrame: string = ``
```

Reply:
For static variables, if you have an initialization value you don't need to type it. TS will infer it's a string here.